examples/flutter_gallery/lib/welcome/steps/explore_step.dart

```diff
@@ -0,0 +1,163 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gallery/welcome/step.dart';
+
+const String _kGalleryAssetsPackage = 'flutter_gallery_assets';
+
+class PopPosition {
+  PopPosition(
+      {this.top, this.right, this.bottom, this.left, @required this.child});
+  final double top;
+  final double right;
+  final double bottom;
+  final double left;
+  final Widget child;
+}
+
+class ExploreWelcomeStep extends WelcomeStep {
+  ExploreWelcomeStep({TickerProvider tickerProvider})
+      : super(tickerProvider: tickerProvider);
+
+  @override
+  String title() => 'Explore Flutter!';
+  @override
+  String subtitle() =>
+      'Start being highly productive and do more with less code. Explore what you can do with Flutter!';
+
+  @override
+  Widget imageWidget() {
+    _setupAnimations();
```

Comment:
In general, favor creating Widgets (rather that classes that are widget-building-function containers) so you don't have to recreate everything on every frame.